src/App.tsx

```diff
@@ -1,24 +1,54 @@
 import React from 'react';
 import './App.scss';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+import { Clock } from './components/Clock';
+
+type State = {
+  isClock: boolean;
 };
 
+class App extends React.Component<{}, State> {
+  state: State = {
+    isClock: true,
+  };
+
+  isClockSet(isVisible: boolean): void {
+    this.setState({ isClock: isVisible });
+  }
+
+  render() {
+    const { isClock } = this.state;
+
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+
+        <button
+          className="App__button"
+          type="button"
+          onClick={() => this.isClockSet(true)}
+        >
+          Show Clock
+        </button>
```

Comment:
What will be if you press double time on this button? (look more detailed to console)